spock-specs/src/test/groovy/org/spockframework/smoke/extension/PendingFeatureExtensionSpec.groovy

```diff
@@ -61,6 +84,52 @@ def bar() {
     }
   }
 
+  def "@PendingFeature includes last reason in exception message if applied twice"() {
+    when:
+    def result = runner.runSpecBody """
+@PendingFeature(reason='42')
+@PendingFeature(reason='4711')
```

Comment:
This looks even more like an error

Reply:
While there is probably no sensible use-case for this in the wild, the test is just intended to ensure a stable reason being shown in the error message. As for the redundancy-check or whatever, let's continue above.